src/Form/MultiSelect.php

```diff
@@ -4,15 +4,43 @@
 
 namespace MyParcelCom\JsonSchema\FormBuilder\Form;
 
-use MyParcelCom\JsonSchema\FormBuilder\Properties\Items\Items;
-use MyParcelCom\JsonSchema\FormBuilder\Properties\Meta\Meta;
-use MyParcelCom\JsonSchema\FormBuilder\Properties\Meta\MetaFieldType;
-use MyParcelCom\JsonSchema\FormBuilder\Properties\SchemaProperty;
-use MyParcelCom\JsonSchema\FormBuilder\Properties\SchemaPropertyType;
+use InvalidArgumentException;
+use MyParcelCom\JsonSchema\FormBuilder\SchemaProperties\Items\Items;
+use MyParcelCom\JsonSchema\FormBuilder\SchemaProperties\Meta\Meta;
+use MyParcelCom\JsonSchema\FormBuilder\SchemaProperties\Meta\MetaFieldType;
+use MyParcelCom\JsonSchema\FormBuilder\SchemaProperties\SchemaProperty;
+use MyParcelCom\JsonSchema\FormBuilder\SchemaProperties\SchemaPropertyType;
+use MyParcelCom\JsonSchema\FormBuilder\Translations\LabelTranslationCollection;
 use Override;
 
 class MultiSelect extends ChoiceField
 {
+    /**
+     * @param array<string>|null $initialValue
+     */
+    public function __construct(
+        string $name,
+        string $label,
+        OptionCollection $options,
+        bool $isRequired = false,
+        ?string $help = null,
+        ?LabelTranslationCollection $labelTranslations = null,
+        protected ?array $initialValue = null,
```

Comment:
Is there a reason for _protected_ instead of _private_? Furthermore, I'd add _readonly_here too:
```suggestion
        private readonly ?array $initialValue = null,
```